core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -64,8 +64,7 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
   }
 
   protected def isNewGroupCoordinatorEnabled: Boolean = {
-    cluster.config.serverProperties.get("group.coordinator.new.enable") == "true" ||
-      cluster.config.serverProperties.get("group.coordinator.rebalance.protocols").contains("consumer")
+    cluster.config.serverProperties.get("group.coordinator.new.enable") == "true"
```

Comment:
I guess the auto-enable will be removed from production later?

https://github.com/apache/kafka/blob/a537e716eb4c168373c0c9d361e68382a63ee637/core/src/main/scala/kafka/server/KafkaConfig.scala#L595

Reply:
Correct. I have a second PR doing the config changes.